exporter/sumologicexporter/config.go

```diff
@@ -37,6 +37,7 @@ type Config struct {
 	// Format to post logs into Sumo. (default json)
 	//   * text - Logs will appear in Sumo Logic in text format.
 	//   * json - Logs will appear in Sumo Logic in json format.
+	//   * otlp - Logs will be send in otlp format and will appear in Sumo Logic in text format.
```

Comment:
This isn't correct. Rather, the way the logs are presented in Sumo will depend on the record attributes and the data type of Body. 
* If record attributes are empty, and the body is a string, we show plain text. 
* If record attributes are empty, and the body is a structured data type, we show the JSON equivalent of body.
* If they're not, we show a JSON object with the log body in the "body" key, and the rest of record attributes as the rest of the keys.

To be honest, this is complicated enough that it should really be a README section, or we can link to the Sumo documentation.

Reply:
I linked doucumentation in `README.md` and `config.go`